src/tir/schedule/primitive/blockize_tensorize.cc

```diff
@@ -427,7 +430,8 @@ Stmt MakeLoopNest(Stmt stmt, const std::vector<const ForNode*>& loops) {
 }
 
 BlockRealize BlockizeImpl(const ScheduleState& self, const StmtSRef& loop_sref,
-                          Map<Block, Block>* block_sref_reuse, arith::Analyzer* analyzer) {
+                          bool preserve_unit_iters, Map<Block, Block>* block_sref_reuse,
```

Comment:
nit: put preserve_unit_iters at the end of argument like other functions with preserve_unit_iters introduced.